src/pages/home/live-pricing/index.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { useState } from 'react'
+import styled, { css } from 'styled-components'
+import LiveMarketTable from './components/_live_market_table'
+import { TAvailableLiveMarkets } from './_types'
+import { market_buttons } from './_utils'
+import { Flex } from 'components/containers'
+import { Header } from 'components/elements'
+import device from 'themes/device'
+import { Button } from 'components/form'
+import { Localize } from 'components/localization'
+import useAuthCheck from 'components/hooks/use-auth-check'
+import useHandleLogin from 'components/hooks/use-handle-login'
+import { handleRedirectToTradersHub } from 'components/layout/nav/util/nav-methods'
+
+const LivePricingSection = styled.section`
+    background-color: var(--color-white);
+    min-height: 780px;
+    display: flex;
+    flex-direction: column;
+    padding: 40px 0 80px;
+`
+
+const ContainerWrapper = styled(Flex)`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 36px;
+    margin: 0 auto;
+    padding: 2rem;
+    width: 60%;
+    @media ${device.tablet} {
+        width: 100vw;
+    }
+`
+
+const Markets = styled.div`
+    position: relative;
+    width: 100%;
+`
+
+const MarketsContainer = styled.div`
+    margin: 0 auto;
+    overflow-y: hidden;
+    overflow-x: auto;
+    justify-content: center;
+    align-items: center;
+    display: flex;
+    padding: 2rem;
+    scrollbar-width: none; /* Firefox */
+    ::-webkit-scrollbar {
+        display: none;
+    }
+    @media ${device.tablet} {
+        justify-content: flex-start;
+    }
+`
+const MarketButton = styled.button<{ selected: boolean }>`
+    margin: 0;
+    min-height: 48px;
+    position: relative;
+    border-bottom: 2px solid red;
+    background: none;
+    border: none;
+    padding: 16px 40px;
+    cursor: pointer;
+    transition: all 0.1s ease-in;
+    white-space: nowrap;
+
+    ${Header} {
+        font-weight: normal;
+        font-size: 2rem;
```

Comment:
Do we need this param in case when `<Header type="paragraph-2">` already have font-size?